ports/ot/port.json

```
@@ -0,0 +1,37 @@
+{
+  "version": 4,
+  "name": "ot.zip",
+  "items": [
+    "Ot.sh",
+    "ot"
+  ],
+  "items_opt": null,
+  "attr": {
+    "title": "Ot",
+    "porter": [
+      "tabreturn"
+    ],
+    "desc": "Autumn arrived; now Ot must gather the leaves. A 2D grid-based puzzle game with 22 levels.",
+    "desc_md": null,
+    "inst": "Ready to run.",
+    "inst_md": null,
+    "genres": [
+      "puzzle"
+    ],
+    "image": {},
+    "rtr": true,
+    "exp": false,
+    "runtime": [],
+    "store": [
+      {
+        "name": "Koliao",
```

Afaik this should be the name of the store, not the developer. I think at some point we intended this to be machine parsable for future expansion (e.g. auto downloaders, ratings, etc...)

```suggestion
        "name": "Itch",
```